Blog/app/models/post.rb

```diff
@@ -0,0 +1,20 @@
+require 'date'
+class Post < ApplicationRecord
+
+    validates :title, presence: true
+
+    validates :body,presence: true, length:{minimum: 50}
+   
+    def formatted_time
```

Comment:
I dont think there is any point (right now) in having this method as you only use it in your view:
`time_ago_in_words(post.formatted_time)`
when I think that 
`time_ago_in_words(post.created_at)` should work just fine.